app/api/views.py

```diff
@@ -13,5 +13,11 @@ def process_coords():
     Returns: This returns a post request of create_segments().
     """
 
-    data = json.loads(request.form.get("coords")).get("b")
-    return "{}".format(create_segments(data))
+    data = json.loads(request.form.get("coords"))
+    count = 1;
```

Comment:
Maybe reset count back to 1 when this endpoint is called again because it will cause duplicate data.